JobRunner/Callback.py

```diff
@@ -12,7 +12,9 @@ class Callback():
     def __init__(self):
         workdir = os.environ.get("JOB_DIR", '/tmp/')
         self.conf = Config(job_id="callback", workdir=workdir, use_ee2=False)
-        self.ip = os.environ.get('CALLBACK_IP', get_ip())
+        self.ip = os.environ.get('CALLBACK_IP')
```

Comment:
Would `self.ip = os.environ.get('CALLBACK_IP') or get_ip()` work?

Reply:
👍 sure. It works exactly the same way. More cleaner.